### PR TITLE
ui: Temporarily force all UI capabilities

### PR DESCRIPTION
**Pull request by johncowen**
Fixes https://github.com/hashicorp/consul/issues/11511

Related: https://github.com/hashicorp/consul/issues/11098

cc @jkirschner-hashicorp @dnephin

### CODE REVIEW

File: ui/packages/consul-ui/app/services/repository/permission.js [hunk 1]
```diff
@@ -148,6 +148,16 @@ export default class PermissionService extends RepositoryService {
   async findAll(params) {
     params.resources = REQUIRED_PERMISSIONS;
     this.permissions = await this.findByPermissions(params);
+    /**/
+    // Temporarily revert to pre-1.10 UI functionality by overwriting frontend
+    // permissions. These are used to hide certain UI elements, but they are
+    // still enforced on the backend.
```
Comment (evrowe) [inline ui/packages/consul-ui/app/services/repository/permission.js:154]: Appreciate the comment for context. I know that the first word in the comment is "Temporarily," but I wonder if we should be a little more explicit about the fact that we intend to return to the intended behavior once some back-end dependencies have been resolved? We don't need to commit to a specific time frame but I feel like making the intentions clear could be beneficial to external contributors.

**Review by evrowe — APPROVED** [review]
Looks good from here, left a non-blocking question .

File: ui/packages/consul-ui/app/services/repository/permission.js (hunk 1 above)
Comment (jkirschner-hashicorp) [reply ui/packages/consul-ui/app/services/repository/permission.js:154]: We could mention GH-11098 specifically as the issue to follow for a longer-term fix.

Comment (johncowen) [reply ui/packages/consul-ui/app/services/repository/permission.js:154]: Added some more detail here inc. GH issue link

Comment (evrowe) [reply ui/packages/consul-ui/app/services/repository/permission.js:154]: :shipit:

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/498950.

**Pull request merged**